app/components/message-chat/component.js

```diff
@@ -7,16 +7,28 @@ export default Ember.Component.extend({
 
   formattedContent: function() {
     const content = this.get('message.content');
+    let out;
 
-    let out = linkifyStr(content, {
-      linkAttributes: { rel: 'nofollow' },
-      validate: {
-        url: function (value) {
-          return /^(http)s?:\/\//.test(value);
+
+    // Images
+    if (content.match(/^(http)s?:\/\/.*(gif|png|jpg|jpeg)$/)) {
```

Comment:
Shouldn't the regex be case insensitive so extensions like `.JPG` would work as well?